openhtf/io/proto/test_runs.proto

```diff
@@ -93,61 +210,96 @@ message TestRun {
   // the test or otherwise responsible for the data.
   optional string operator_name = 22;
 
-  // Not supported by OpenHTF
   // For non-serialized items, we can track them via lot_number and a part_id
   // within the lot (such as "This try of items is lot #FOT123 and this is the
   // part in slot 6 of the tray.  In this case, a unique dut should be
   // synthesized and stored in the required field dut_serial, and the
   // synthetic_dut flag should be set.
-  optional string lot_number = 23 [deprecated=true];
-  optional string part_id = 24 [deprecated=true];
-  optional bool synthetic_dut = 25 [deprecated=true];
+  optional string lot_number = 23;
+  optional string part_id = 24;
+  optional bool synthetic_dut = 25;
 
-  //optional Assembly assembly = 26;
-  repeated Timing timings = 27 [deprecated=true];
-  repeated Phase phases = 28 [deprecated=true];
+  optional Assembly assembly = 26;
```

Comment:
I do want to exclude the assembly file, though, it's an entire proto file of unused stuff, and the one-line delta of having this line commented out isn't unmanageable.